projects/1.21/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -0,0 +1,2009 @@
+  "ars_nouveau.page1.spell_mana": "使用法术书施法时需要消耗魔力。穿戴特殊的魔法盔甲，或是为装备附上魔力提升、魔力再生等魔咒，均可提升玩家的最大魔力和魔力恢复速度。另外，每在法术书中解锁一个构件，均会给予额外的最大魔力和魔力恢复速度。",
+  "ars_nouveau.page1.starbuncle_charm": "星宝石兽会自然出现在林地区域四处寻找金粒。虽然一般来说星宝石兽害怕人类，但它们会让持有金粒的人靠近。在捡起金粒后，星宝石兽即会从世界上消失，并留下一份星宝石兽信物。",
+  "ars_nouveau.page1.starting_automation": "可借助法术炮塔在自动化中使用法术。自动收割机、树木农场、采石场、蛋糕农场、玻璃工厂，如此种种都可用它完成！而到物品运输、自动合成、资源生成等，参阅护符召唤而来的多种魔法实体。",
+  "ars_nouveau.page1.storage_lectern": "存储讲台可用于查看、管理多个相连的容器，也可使用其中物品进行合成。可连接容器的数量上限取决于绑定至讲台的书龙数量。对讲台使用书龙护符即可新添书龙。先对容器使用支配之杖，再对讲台使用，即可建立或断开连接。允许连接30格内的容器。",
+  "ars_nouveau.page1.summon_bed": "装饰性的床铺。可使用支配之杖将星宝石兽绑定至魔花床，它们便会在空闲时间到床上休息。",
+  "ars_nouveau.page1.summoning_familiars": "在召唤使魔之前，需要先准备好所召唤实体的绑定手稿。在相应实体旁施展绑定仪式即可获得绑定手稿。有关实体的完整列表请参阅使魔相关条目。获得绑定手稿后，使用即可习得该使魔。",
+  "ars_nouveau.page1.trinkets": "一部分物品和饰品可以增强施法能力，或提供独特的增益。对于强化施法能力，可以选择节魔之戒或魔力再生护身符。对于长途跋涉，可以选择浮空腰带。还可考虑使用湮灭之瓶提升采矿效率。",
+  "ars_nouveau.page1.upgrades": "2级和3级的魔符分别需要借助法师法术书和大法师法术书才能使用。高等级的法术书可让你施放高等级的法术，在自动化中使用有关法术，还可额外提供最大魔力和魔力恢复速度。升级好法术书后，可以同时升级魔法盔甲，如此也能增长施法能力。",
+  "ars_nouveau.page1.volcanic_sourcelink": "火山魔源通道会消耗可燃烧的物品生成魔源。至高木原木会额外产出魔源，其中以烈焰至高木为最多。火山魔源通道产生魔源的过程中也会产生热量，能把石头变为岩浆块，再变为熔岩。火山魔源通道会向附加的魔源罐输出魔源，距离最近的优先。",
+  "ars_nouveau.page1.warp_portal": "折跃传送门与折跃卷轴类似，均可向同维度内任意位置进行单向传送。搭建折跃传送门前，需用魔源石或其变种构成一个长方形，并在附近放下填满的魔源罐。然后，向框架内投入一份已绑定位置的折跃卷轴。若魔源足够，即会创建折跃传送门。",
+  "ars_nouveau.page1.weald_waddler": "林地行者死亡后会变为林地蹒跚者。林地蹒跚者会缓慢长成林地行者，施加骨粉可以加速。在长回林地行者之前，林地蹒跚者不会作战，也不具备自保能力。",
+  "ars_nouveau.page1.weald_walker": "林地行者是会自行移动的至高木树，它们会守卫家园，阻止敌对生物进犯。林地行者可由启蒙仪式制造。召唤后的林地行者会无目的地四处漫步，直至为其指定家园。如需为林地行者指定家园，应先对林地行者使用支配之杖，再对希望它驻守的方块使用。",
+  "ars_nouveau.page1.whirlisprig_charm": "风转草是充满好奇心的森林精灵，只出现于森林区域。召唤出的风转草可以指定世界中的家园，之后它们会根据周围的自然方块产出木材、农作物、种子、花朵等自然资源。在野生风转草旁让一棵树长成可向其表示友好，此时它们会掉落风转草信物。",
+  "ars_nouveau.page1.wilden": "荒野族裔是一类敌对生物，常在夜晚出没于荒野兽穴附近。荒野守卫者只会出现在寒冷的生物群系中，荒野猎手和荒野伏击者则出没于任意森林类生物群系。",
+  "ars_nouveau.page1.wixie_charm": "薇克精可以消耗魔源自动合成物品。对生命值在一半或以下的女巫施放“解除魔法”，即可获得薇克精信物。获得薇克精护符后，对炼药锅使用即可召唤薇克精。薇克精可以染成红色、白色、黑色、蓝色、绿色。",
+  "ars_nouveau.page1.world_generation": "至高木树有多种装饰性变种，也需用来合成若干重要方块。至高木树上结出的果实可以用来酿造药水。出现于针叶林的魔源浆果，则是酿造魔力再生药水、为菌丝魔源通道供能的必需品。世界中还会生成多种生物，可通过占卜杖、或2级法术书与“感知魔法”令其现身。",
+  "ars_nouveau.page2.alakarkinos_charm": "喜欢随着附近的唱片机跳舞。",
+  "ars_nouveau.page2.amethyst_golem_charm": "",
+  "ars_nouveau.page2.applying_perks": "纤维只在整个盔甲套装内起效一次，且同种纤维不会叠加。更多有关安装纤维的信息请参阅改衣台条目。",
+  "ars_nouveau.page2.armor_upgrading": "盔甲自身也具有3个等级，可使用附魔装置和有关条目中的升级配方升级盔甲。各等级均会增加盔甲的魔力恢复速度增益，同时增加纤维槽的数量和等级。",
+  "ars_nouveau.page2.basic_spell_turret": "附魔法术炮塔施法的消耗是基础炮塔的一半。",
+  "ars_nouveau.page2.bookwyrm_charm": "书龙死亡或被“解除魔法”时会掉落为护符。",
+  "ars_nouveau.page2.drygmy_charm": "德格米还能产出经验宝石！",
+  "ars_nouveau.page2.how_to_enchant": "要附上II级或更高级的魔咒，所给物品必先拥有前一级的魔咒。例如，亡灵杀手 III要求所给物品拥有亡灵杀手 II。",
+  "ars_nouveau.page2.illusion_blocks": "虚幻编织块的性质与镜像编织块一致，但会像空气一样不阻拦实体。“感知魔法”可让此类方块隐形，以揭示隐藏的通道。",
+  "ars_nouveau.page2.item_detector": "若给予展示盒过滤卷轴，则展示盒会按照该过滤器计数。",

```

```suggestion
  "ars_nouveau.page2.item_detector": "若给予展示盒过滤卷轴，则展示盒会按照该过滤器匹配的物品计数。",

```
我看这个中文有点没理解，还是按照原文全译吧。这样跟容易理解一些。

---

换成“则展示盒会按照该过滤器统计物品数量”不知道怎么样